lsp-mode.el

```diff
@@ -5103,10 +5106,24 @@ It will show up only if current point has signature help."
       (funcall action-handler action)
     (lsp--send-execute-command command arguments?)))
 
-(lsp-defun lsp-execute-code-action ((action &as &CodeAction :command? :edit?))
+(defun lsp-execute-code-action (action)
   "Execute code action ACTION.
-If ACTION is not set it will be selected from `lsp-code-actions-at-point'."
+If ACTION is not set it will be selected from `lsp-code-actions-at-point'.
+Request codeAction/resolve for more info if server supports."
   (interactive (list (lsp--select-action (lsp-code-actions-at-point))))
+  (if (-> (lsp--server-capabilities)
```

Comment:
shouldn't we perform resolve only if the property we are looking for is not present. This is how resolve works for code lenses if memory serves me right.

Reply:
Yeah, I thought about that too but I was not sure, in this case we should check if `command` or `workspaceEdit` is present, right?

Reply:
I think it is edit not workspaceEdit but yes.